### PR TITLE
refactor: rendering events from JSON  #22

### PR DESCRIPTION
**Pull request by mosespace**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
> This repository replaces the hard-coded event data in the Card component with the imported data, from the `../content/events.json`

Fixes Issue #22 

## Type of change

Please chose options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Firmware version:
- Hardware:
- Toolchain:
- pnpm:

# Checklist:

- [x] I have read the [Contributing Guidelines](https://github.com/OSCA-Kampala-Chapter/react-ui/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run pnpm run test to check if all tests are passing, and/or pnpm run test -- -u to update snapshots if I created and/or updated React Components.
- [x] I have checked that the build works locally and that npm run build work fine.
- [ ] I've covered new added functionality with unit tests if necessary.
- [x] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](https://github.com/airqo-platform/AirQo-frontend/blob/staging/.github/PULL_REQUEST_TEMPLATE.md#what-are-the-relevant-tickets)" section to [link the issue.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by mosespace** [comment]
@HassanBahati I think I might need to import this in this way: 

```js
  import * as events from "../content/events.json"
```

and try to access it like this:

```css
  <div className="font-bold text-xl mb-2">{Events.name}</div>
```

**Comment by HassanBahati** [comment]
@mosespace thank you for the great work.

let me look into this

**Comment by mosespace** [comment]
@HassanBahati, what's the progress on this?

**Comment by mosespace** [comment]
@HassanBahati what went wrong?

**Comment by HassanBahati** [comment]
> @HassanBahati what went wrong?

Hey @mosespace , sorry been quite caught up...nevertheless I will find time this evening to push the changes.

Happy Easter 🎆 🍾

**Comment by HassanBahati** [comment]
@mosespace i think this pr is now ready...you may have a look at the changes

**Review by mosespace — COMMENTED** [review]
I think we are good to go!!!

File: pages/index.tsx [hunk 1]
```diff
@@ -4,7 +4,14 @@ import Image from 'next/image'
...
 const Home: NextPage = () => {
-  function Card(props: { events: any }) {
+  interface ICardProps {
+    partnersData: {
+      name: string
+      description: string
+      tags: string[]
+    }
+  }
+  function Card(props: ICardProps) {
```
[the remark this answers is not in the document]
Comment (HassanBahati) [reply pages/index.tsx]: @mosespace for the particular part youve spotted, you can refer here https://www.typescriptlang.org/docs/handbook/interfaces.html

**Pull request merged**